### PR TITLE
add contact avatar feature

### LINKED ISSUE #12: (feat): To load photos?
# Feature request

Is it possible to load contact's photo?

### PR DESCRIPTION
**Pull request by SpeakSugar**

### CODE REVIEW

File: src/android/ContactsX.java [hunk 1]
```diff
@@ -24,10 +24,7 @@
 import org.json.JSONException;
 import org.json.JSONObject;
 
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.Locale;
-import java.util.Map;
+import java.util.*;
```
Comment (EinfachHans) [inline src/android/ContactsX.java:27]: Why are you importing the whole **java.util** here?

**Review by EinfachHans — CHANGES_REQUESTED** [review]
- Add `avatar` to definition File
- Add it to Documentation with a Usage example
- If we add a feature to save a contact with an avatar, we should make sure to be able to edit the avatar of an existing contact as well
- If we add a Feature to save a contact with an avatar we also should have the feature to include the avatars in the find method

File: src/android/ContactsX.java (hunk 1 above)
Comment (SpeakSugar) [reply src/android/ContactsX.java:27]: maybe I import base64 util, which caused the idea editor merge it to '*'. Does this affect performance?

**Comment by SpeakSugar** [comment]
> * Add `avatar` to definition File
> * Add it to Documentation with a Usage example
> * If we add a feature to save a contact with an avatar, we should make sure to be able to edit the avatar of an existing contact as well
> * If we add a Feature to save a contact with an avatar we also should have the feature to include the avatars in the find method

If I have time, I will finish it later

File: src/android/ContactsX.java (hunk 1 above)
Comment (EinfachHans) [reply src/android/ContactsX.java:27]: I don't think so, i just don't like it. I think importing the single Classes gives a better overview

Comment (SpeakSugar) [reply src/android/ContactsX.java:27]: ok, I will fix it and pull new request again

**Pull request closed without merging**